Rover/sailboat.h

```diff
@@ -20,11 +20,23 @@ class Sailboat
 {
 public:
 
+    enum class OPTION : uint32_t {
+        GET_OUT_OF_IRONS_IN_AUTO =  (1U << 0U),
+    };
+
+    bool option_is_set(OPTION option)const
+    {
+        return (static_cast<uint32_t>(options.get()) & static_cast<uint32_t>(option)) != 0U;
```

Comment:
```suggestion
        return (uint32_t(options.get()) & uint32_t(option)) != 0U;
```
Normal cast should be sufficient.

Reply:
"Normal cast" ? What is a Normal cast?

I refer you to the C++ Authors core guidelines https://github.com/isocpp/CppCoreGuidelines